client/jetpack-connect/remote-credentials.js

```diff
@@ -154,8 +154,8 @@ export class OrgCredentialsForm extends Component {
 						value={ username || '' }
 					/>
 				</div>
-
 				<div className="jetpack-connect__password-container">
+					{ isSubmitting ? <Spinner className="jetpack-connect__creds-form-spinner" /> : null }
```

Comment:
This is fine, but if you want you can instead do 
```
{ isSubmitting && <Spinner className="jetpack-connect__creds-form-spinner" /> }
```

Reply:
true! thanks